src/google-classroom-extractor/google_classroom_extractor/config.py

```diff
@@ -14,13 +14,29 @@
 
 
 def _is_running_in_notebook() -> bool:
+    """
+    Determine whether code is running in a Jupyter Notebook
+
+    Returns
+    -------
+    bool
+        True if code is running in a Jupyter Notebook
+    """
     main = __import__("__main__", None, None, fromlist=["__file__"])
     return not hasattr(main, "__file__")
 
 
 def get_sync_db_engine() -> sqlalchemy.engine.base.Engine:
+    """
+    Create a SQL Alchemy Engine for a SQLite file
+
+    Returns
+    -------
+    sqlalchemy.engine.base.Engine
+        a SQL Alchemy Engine
+    """
     running_in_notebook: bool = _is_running_in_notebook()
-    logging.debug(f"Running in Jupyter Notebook: {running_in_notebook}")
+    logging.debug("Running in Jupyter Notebook: %s", running_in_notebook)
```

Comment:
While I am not concerned by this change, I do find it a bit odd...

Reply:
Yes, don't like the ugliness, but I've seen that some people prefer to stick with logger's old style because in general it will still log if the expression causes an exception.

Reply:
e.g., if running `logging.debug("something %s", some_call_causes_exception()` then at least we still get a log output that says "something"?

Reply:
Yes, we should get log formatter stuff plus "something" plus a message from logger that an exception was thrown during output.